HashRegistrarSimplified.sol

```diff
@@ -388,14 +388,21 @@ contract Registrar {
         entry h = _entries[_hash];
 
         h.value =  max(h.value, minPrice);
+        h.deed.setBalance(h.value, true);
 
-        // Assign the owner in ENS, if we're still the registrar
-        if(ens.owner(rootNode) == address(this))
-            ens.setSubnodeOwner(rootNode, _hash, h.deed.owner());
+        if (h.value >= 1000 ether) {
+            // Temporary registrar will not accept high value names yet
+            h.deed.closeDeed(995);
+            h.deed = Deed(0);
```

Comment:
No, I was asking how this will affect existing behaviour; previously you could only invalidate after an auction, while this makes a name invalid _during_ an auction. It looks like this will result in revealBid rejecting any attempts to reveal any later bids, therefore locking up and eventually throwing away everyone else's ether.